<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="42595" started-at="2021-08-17T04:43:29Z" finished-at="2021-08-17T04:44:11Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="42595" started-at="2021-08-17T04:43:29Z" finished-at="2021-08-17T04:44:11Z">
      <class name="com.BookingCommerce.qa.TestCases.LoginPageTest">
        <test-method status="PASS" signature="Setup()[pri:0, instance:com.BookingCommerce.qa.TestCases.LoginPageTest@1753acfe]" name="Setup" is-config="true" duration-ms="16024" started-at="2021-08-17T10:13:29Z" finished-at="2021-08-17T10:13:45Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Setup -->
        <test-method status="PASS" signature="LoginPageTitleTest()[pri:0, instance:com.BookingCommerce.qa.TestCases.LoginPageTest@1753acfe]" name="LoginPageTitleTest" duration-ms="118" started-at="2021-08-17T10:13:45Z" finished-at="2021-08-17T10:13:45Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginPageTitleTest -->
        <test-method status="PASS" signature="teardown()[pri:0, instance:com.BookingCommerce.qa.TestCases.LoginPageTest@1753acfe]" name="teardown" is-config="true" duration-ms="637" started-at="2021-08-17T10:13:45Z" finished-at="2021-08-17T10:13:45Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
        <test-method status="PASS" signature="Setup()[pri:0, instance:com.BookingCommerce.qa.TestCases.LoginPageTest@1753acfe]" name="Setup" is-config="true" duration-ms="24264" started-at="2021-08-17T10:13:45Z" finished-at="2021-08-17T10:14:10Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Setup -->
        <test-method status="FAIL" signature="logintest()[pri:0, instance:com.BookingCommerce.qa.TestCases.LoginPageTest@1753acfe]" name="logintest" duration-ms="835" started-at="2021-08-17T10:14:10Z" finished-at="2021-08-17T10:14:10Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Login] but found [Dashbord]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Login] but found [Dashbord]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:176)
	at org.testng.Assert.assertEquals(Assert.java:186)
	at com.BookingCommerce.qa.TestCases.LoginPageTest.logintest(LoginPageTest.java:40)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logintest -->
        <test-method status="PASS" signature="teardown()[pri:0, instance:com.BookingCommerce.qa.TestCases.LoginPageTest@1753acfe]" name="teardown" is-config="true" duration-ms="701" started-at="2021-08-17T10:14:10Z" finished-at="2021-08-17T10:14:11Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- com.BookingCommerce.qa.TestCases.LoginPageTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
